gateway/modules/crud/sql/describe.go

```diff
@@ -50,6 +54,10 @@ CASE
     WHEN t.constraint_type = 'UNIQUE' THEN 'UNI'
     ELSE ''
 END AS "Key",
+CASE 
+	WHEN isc.data_type = 'bigint' AND t.constraint_type = 'PRIMARY KEY' THEN 'true'
```

Comment:
What if someone had used serial?

Reply:
Altaf has sent me a new query for Postgres & SQL server. I have to test that out

Reply:
Okay. So I won't merge this PR till then